download.sh

```diff
@@ -28,17 +28,17 @@ popd
 
 
 install_fabric_manager () {
-    local driver_ver="${DRIVER_VERSION}"
+    FABRIC_MANAGER_VERSION="${DRIVER_VERSION}"
     if [[ "${DRIVER_VERSION}" == "535.161.08" ]]; then
-      local driver_ver="535.104.05"
-      # Temporary as latest fabric manager version here is not the same as the driver version. https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/
-      curl -fsSLO https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/fabricmanager-linux-x86_64-${driver_ver}-archive.tar.xz
+      FABRIC_MANAGER_VERSION="535.104.05"
+      # TODO Temporary as latest fabric manager version here is not the same as the driver version. https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/
+      curl -fsSLO https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}-archive.tar.xz
```

Comment:
do you need to duplicate the `curl` code ? seems like it could be outside the `if`

Reply:
Moved